src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsrestrictedpatientsapi/services/DomainEventSubscriber.kt

```diff
@@ -29,13 +29,12 @@ class DomainEventSubscriber(
     val event = gson.fromJson(requestJson, Event::class.java)
     val prisonerReceivedEvent = gson.fromJson(event.Message, PrisonerReceivedEvent::class.java)
 
-    log.info("Domain event received: {}", prisonerReceivedEvent.eventType)
```

Comment:
weren't logging anything interesting here, just the type of event.  Think it was used initially for testing to ensure that the published event can then be consumed, but just adds more complexity.